### PR TITLE
Minimal documentation

### PR DESCRIPTION
**Pull request by MiaAltieri**
Added:
- Overview
- Requirements
- Configurations

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,30 @@
 # Charmed MongoDB Operator
 
 
-## Description
+## Overview
 
-The Charmed MongoDB Operator delivers automated operations management from day 0 to day 2 on the [MongoDB Community Edition](https://github.com/mongodb/mongo) document database.
```
[the remark this answers is not in the document]
Comment (michtabirao) [reply README.md:6]: But I guess we have to wait to implement these changes once Jared published MongoDB Percona OCI image and snaps

File: README.md [hunk 2]
```diff
@@ -1,12 +1,30 @@
...
+The Charmed MongoDB Operator delivers automated operations management from day 0 to day 2 on the [MongoDB Community Edition](https://github.com/mongodb/mongo) document database. It is an open source, end-to-end, production ready data platform on top of cloud native technologies.
 
-MongoDB is a general purpose distributed document database. This operator charm deploys and operates MongoDB (vs 5.0) on physical or virtual machines as a single replica.
+MongoDB is a popular NoSQL database application. It stores its data with JSON-like documents creating a flexible experience for users; with easy to use data aggregation for data analytics. It is a distributed database, so vertical and horizontal scaling come naturally.
 
+This operator charm deploys and operates MongoDB on physical or virtual machines. It offers features such as replication, TLS, password rotation, and easy to use integration with applications. The Charmed MongoDB Operator meets the need of deploying MongoDB in a structured and consistent manner while allowing the user flexibility in configuration. It simplifies deployment, scaling, configuration and management of MongoDB in production at scale in a reliable way.
+
```
[the remark this answers is not in the document]
Comment (michtabirao) [reply README.md:11]: But I guess we have to wait to implement these changes once Jared published MongoDB Percona OCI image and snaps

File: README.md [hunk 3]
```diff
@@ -1,12 +1,30 @@
...
+auto-delete - `boolean`; When a relation is removed, auto-delete ensures that any relevant databases
+associated with the relation are also removed. Set with `juju config mongodb auto-delete=<bool>`.
+
+admin-password - `string`; The password for the database admin. Set with `juju run-action mongodb/leader set-admin-password --wait`
+
+tls external key - `string`; TLS external key for encryption outside the cluster. Set with `juju run-action mongodb/0 set-tls-private-key "external-key=$(base64 -w0 external-key-0.pem)" --wait`
+
+tls internal key - `string`;  TLS external key for encryption inside the cluster. Set with `juju run-action mongodb/0 set-tls-private-key "internal-key=$(base64 -w0 internal-key.pem)"  --wait`
 
 ## Usage
```
[the remark this answers is not in the document]
Comment (michtabirao) [reply README.md:30]: In the future, once we have Percona as artefact

## License
The Charmed MongoDB Operator is distributed under the Apache Software License, version 2.0. It [installs/operates/depends on] [Percona Server for MongoDB](https://github.com/percona/percona-server-mongodb), which is licensed under the Server Side Public License (SSPL)

## Trademark notice

‘MongoDB' is a trademark or registered trademark of MongoDB Inc. Other trademarks are property of their respective owners.

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (MiaAltieri) [reply README.md:11]: Yes this is a great suggestion. Let's definitely do this after Jared publishes the MongoDB Percona OCI image and snaps and the charm uses those.

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (MiaAltieri) [reply README.md:6]: Yes this is a great suggestion.I'll remember to do this after Jared publishes the MongoDB Percona OCI image and snaps and the charm uses those.

File: README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (MiaAltieri) [reply README.md:30]: Sounds good, the trademark is already that so I'll update the license accordingly

**Review by michtabirao — APPROVED** [review]

**Review by deusebio — APPROVED** [review]

File: README.md [hunk 4]
```diff
@@ -143,6 +161,10 @@ Please see the [Juju SDK docs](https://juju.is/docs/sdk) for guidelines on enhan
 ## License
 The Charmed MongoDB Operator is free software, distributed under the Apache Software License, version 2.0. See [LICENSE](https://github.com/canonical/mongodb-operator/blob/main/LICENSE) for more information.
```
Comment (delgod) [inline README.md:163]: maybe we need to delete obsolete wording?

**Review by delgod — APPROVED** [review]

**Pull request merged**